app/assets/stylesheets/home/components/psychologist_List/psychologist_List.scss

```diff
@@ -0,0 +1,94 @@
+@import "../variables";
+
+$whiteColor: #ffff;
+.List-container {
```

Comment:
What do you think of using `.list-container` instead, just for consistency's sake?